inc/altis-config/class-qm-output-html-altis-config.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Altis Config data output for HTML pages in the Query Monitor plugin.
+ */
+
+namespace Altis\Dev_Tools;
+
+use function Altis\get_config;
+use QM_Collector;
+use QM_Output_Html;
+
+class QM_Output_Html_Altis_Config extends QM_Output_Html {
+
+	public function __construct( QM_Collector $collector ) {
+		parent::__construct( $collector );
+		add_filter( 'qm/output/panel_menus', [ $this, 'panel_menu' ], 50 );
+	}
+
+	public function output() {
+		$config = get_config()['modules'] ?? [];
+
+		if ( ! $config ) {
+			echo esc_html_x( 'Something went wrong with displaying the Altis config settings.', 'Displaying Altis config info in the Query Monitor plugin', 'altis' );
+			return;
+		}
+		?>
+
+		<?php $this->before_tabular_output(); ?>
+
+		<caption>
+			<h2><?php echo esc_html( $this->name() ); ?></h2>
+		</caption>
+		<thead>
+		<tr>
+			<th><?php echo esc_html__( 'Module', 'altis' ) ?></th>
+			<th><?php echo esc_html__( 'Settings', 'altis' ) ?></th>
```

Comment:
```suggestion
			<th><?php echo esc_html__( 'Merged Settings', 'altis' ) ?></th>
```
I think the title "Merged Settings" will make it clearer that this is the final config output after defaults & overrides have been combined.

Reply:
Hm, why not call that "Final Settings"?  "Merged" also might be a bit confusing.

Reply:
What do you feel might be confusing about "Merged"? I was taking that from what we're doing under the hood, to surface what's happening a bit.

Reply:
Sure, I was thinking merged what with what, and "Final" definitely is clear that these are the settings. Just me really, I'll change :)

Reply:
I get you. I think in context with the 3rd column for "Environment Settings" it should be clear enough. We can always change it later depending on feedback we get during the beta release.